telemetry/definitions/commonDefinitions.json

```diff
@@ -4995,6 +5010,42 @@
                 }
             ]
         },
+        {
+            "name": "cwlLiveTail_Start",
+            "description": "When user starts a new LiveTail command",
+            "metadata": [
+                {
+                    "type": "hasLogEventFilterPattern",
+                    "required": false
+                },
+                {
+                    "type": "logStreamFilterType",
+                    "required": false
+                },
+                {
+                    "type": "sessionAlreadyStarted",
+                    "required": true
+                },
+                {
+                    "type": "source",
+                    "required": true
```

Comment:
can you add result to `cwlLiveTail_Start` that suggests what happened as result of that command? and if applicabl capture error reason for Failed cases.